definitions/npm/react-color_v2.x.x/flow_v0.56.x-/react-color_v2.x.x.js

```diff
@@ -131,16 +139,13 @@ declare module "react-color" {
   ) => void;
 
   declare export type InjectedColorProps = {
-    hex?: string,
-    hsl?: HSLColor,
-    rgb?: RGBColor,
-    onChange?: ColorWrapChangeHandler
-  };
-
-  declare export type ExportedColorProps = {
```

Comment:
Removing `ExportedColorProps` is a breaking change since someone might have imported it and used it in their application. Since I removed the need for it I decided to remove the type annotation too. If this is seen as not good change I can add it back.